test/modules/ModuleLib/ReporterClients/ReportContent.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil.ReporterClients
+{
+    using System;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public sealed class ReportContent
```

Comment:
Create an interface for test result content; which should at least have 2 methods - SetResult(string result) and string GetResultMessage().  For example, for TestOperationResultType.DirectMethod, DirectMethodTestResult is created with tracking id, batch id, and TestOperationResultType provided in constructor.  During test, you just need to concatenate sequence number and HttpStatusCode; and call SetResult.